build.sbt

```diff
@@ -1,5 +1,8 @@
 val catsVersion = "2.6.1"
 
+ThisBuild / tlBaseVersion := "21.8" // TODO
```

Comment:
@armanbilge I noticed this TODO in this PR. Do you remember what was left to do here?

Reply:
I'm afraid I don't remember the exact TODO I had in mind. I remember there was a larger discussion about what versioning scheme you wanted to use, so probably related to that.

Reply:
Oh, that makes sense. I'm still not sure where we want to go with that, although I like the idea of semantic versioning with MiMa determining when there is a transitive breaking change. I think we can merge this without deciding that, for now.

Reply:
Sounds good, I was able to get the MiMa thing going in https://github.com/typelevel/catbird/pull/430 :)

I remember why I disabled snapshots, b/c creds weren't setup yet. Let me re-enable them and then it's good to merge.